hera_mc/cm_hookup.py

```diff
@@ -37,11 +37,13 @@ def __get_connection(self, direction, part, rev, port, session):
         if direction == 'up':      # Going upstream
             for conn in session.query(part_connect.Connections).filter((part_connect.Connections.downstream_part == part) &
                                                                        (part_connect.Connections.down_part_rev == rev)):
+                conn.gps2Time()
```

Comment:
Looking at what the `gps2Time` routines do, I think there is a better solution. We could add methods to the classes like this:

```
@property
def start_date(self):
    return Time(self.start_gpstime, format='gps')
```

to create the `Time` objects on demand without needing to remember to add these conversion calls: it all happens transparently if you access `thingie.start_date`. But again, in the interests of rapid turnaround, I think it is OK to consider that possibility in a separate PR/issue.